### ISSUE TITLE
Allow adjusting trace offset

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#10027]
### Description

Just like we can "Drag anywhere in the gray area below to adjust the imagery offset, or enter the offset values in meters." we should be able to do the same thing with traces.

No matter if we selected "OpenStreetMap GPS traces" in Background > Overlays,
or if we used a URL like https://www.openstreetmap.org/edit?gpx=11042475#map=21/42.16581/-87.77856 .

I am just saying just as one would expect imagery to be perfectly aligned, but sometimes it is not (thus manual alignment is allowed,) the same thing could happen to other items, so adjustment should be allowed on them too.

**Comment by tyrasd** [comment]
I don't think that this makes sense for the "OpenStreetMap GPS traces" overlay: gps points might have inaccuracies, but it's rarely something consistent that could be corrected by an offset.

**Comment by jidanni** [comment]
OK. I suppose GPS traces would not be consistently skewed in any one particular direction. If wobbling would wobble in many directions. Nothing that pulling the whole track back and forth would fix.
